PacketDotNet/Ieee80211/ChannelRadioTapField.cs

```diff
@@ -24,218 +24,223 @@
 using PacketDotNet.Utils;
 using MiscUtil.Conversion;
 
-namespace PacketDotNet
+namespace PacketDotNet.Ieee80211
 {
-    namespace Ieee80211
+    /// <summary>
```

Comment:
I see you fixed the indenting here but not on any other file here. I'd probably pick one approach or another. Imo it would be ideal to change the namespaces in one commit and fix the indenting in another, but its confusing to see this file have its indenting fixed but not the other ones.